build.sh

```diff
@@ -204,7 +277,10 @@ make iso BASE="$MYBASE" RELEASE="$MYRELEASE" ARCH="$MYARCH" ROOTPW_HASH="*" MFSR
 
 # scp to distribution site
 if [ "$UPLOAD" = "YES" ]; then
-	scp "$OUTISO" "$CFG_SSH_REMOTEHOST":"$CFG_SSH_REMOTEPATH"
+	#removed#
+	#scp "$OUTISO" "$CFG_SSH_REMOTEHOST":"$CFG_SSH_REMOTEPATH"
+	# new approach to deal with bad uploads, make sure host is configed in .ssh/config
+	rsync -P -e ssh "$OUTISO" "$CFG_SSH_REMOTEHOST":"$CFG_SSH_REMOTEPATH"/"$1"
```

Comment:
What was the issue here?

Reply:
scp was failing at 70% with disconnected

Reply:
what will rsync do? Retry?

Reply:
[stackoverflow answer](https://stackoverflow.com/a/20257021) says:

> scp basically reads the source file and writes it to the destination. It performs a plain linear copy, locally, or over a network.
> 
> rsync also copies files locally or over a network. But it employs a special [delta transfer algorithm](http://rsync.samba.org/tech_report/) and a few optimizations to make the operation a lot faster.